### PR TITLE
pr_checker: consider LITE CI as a ci requirement

### PR DESCRIPTION
**Pull request by cPhost**
Fixes: https://github.com/nodejs/node-core-utils/issues/225

### CODE REVIEW

File: lib/pr_checker.js [hunk 1]
```diff
@@ -203,7 +203,7 @@ class PRChecker {
       cli.error('No CI runs detected');
       this.CIStatus = false;
       return false;
-    } else if (!ciMap.get(FULL)) {
+    } else if (!ciMap.get(FULL) && !ciMap.get(LITE)) {
       status = false;
       cli.error('No full CI runs detected');
```
Comment (joyeecheung) [inline lib/pr_checker.js]: Can you update this to `No full CI runs or lite CI runs detected`?

Comment (cPhost) [reply lib/pr_checker.js]: Done.

**Comment by vsemozhetbyt** [comment]
Can we make a release with this fix?

**Comment by joyeecheung** [comment]
@vsemozhetbyt Done in https://github.com/nodejs/node-core-utils/releases/tag/v1.12.0

**Pull request merged**